### PR TITLE
removed FormatsDictionary: updated ImageResizeHelper

### PR DESCRIPTION
**Pull request by vorobevamv**

### CODE REVIEW

File: src/Kernel.ImageSupport/Helpers/ImageResizeHelper.cs [hunk 1]
```diff
@@ -48,14 +55,15 @@ public class ImageResizeHelper : IImageResizeHelper
           ImageConverter converter = new ImageConverter();
 
           byteString = (byte[])converter.ConvertTo(newImage, typeof(byte[]));
-          extension = ImageFormats.png;
+          extension = ".png";
```
Comment (MariyaSin) [inline src/Kernel.ImageSupport/Helpers/ImageResizeHelper.cs]: png, why?

File: src/Kernel.ImageSupport/Helpers/ImageResizeHelper.cs [hunk 2]
```diff
@@ -111,14 +119,15 @@ public class ImageResizeHelper : IImageResizeHelper
...
```
Comment (MariyaSin) [inline src/Kernel.ImageSupport/Helpers/ImageResizeHelper.cs]: same
create the const string

File: src/Kernel.ImageSupport/Helpers/ImageResizeHelper.cs (hunk 1 above)
Comment (vorobevamv) [reply src/Kernel.ImageSupport/Helpers/ImageResizeHelper.cs]: added comments

File: src/Kernel.ImageSupport/Helpers/ImageResizeHelper.cs (hunk 2 above)
Comment (vorobevamv) [reply src/Kernel.ImageSupport/Helpers/ImageResizeHelper.cs]: done

**Review by VLiamin — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**